### PR TITLE
scil_merge_json move function into module

### PR DESCRIPTION
**Pull request by arnaudbore**
Next one will be: scil_convert_json_to_xlsx.py

I need some approval about the name of new module and the doc @frheault

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_merge_json.py [hunk 1]
```diff
@@ -10,52 +10,9 @@
...
-            dict_1[key] = _average_dict(dict_1[key])
-        elif isinstance(dict_1[key], list) or np.isscalar(dict_1[key]):
-            new_dict = {}
-            for subkey in dict_1.keys():
-                new_dict[subkey] = {'mean': np.average(dict_1[subkey]),
-                                    'std': np.std(dict_1[subkey])}
-            return new_dict
-
-    return dict_1
+from scilpy.tractanalysis.json_utils import _merge_dict, _average_dict
```
Comment (EmmaRenauld) [inline scripts/scil_merge_json.py:15]: About the names: I was trying to keep the convention that functions with _ are only reused in another function, not called in script. (And I don't add unit tests for these functions).

Are they worth renaming merge_dict and average_dict?

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by EmmaRenauld — APPROVED** [review]
LGTM.

**Pull request merged**